src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -397,23 +398,27 @@ protected final void verify(Checker checker,
      * @param testConfig hardcoded test config.
      * @param parsedConfig parsed config from input file.
      */
-    private static void verifyConfig(Configuration testConfig, Configuration parsedConfig) {
+    private static void verifyConfig(Configuration testConfig, Configuration parsedConfig,
+                                     InputConfiguration inputConfiguration)
+            throws CheckstyleException {
         assertWithMessage("Check name differs from expected.")
                 .that(testConfig.getName())
                 .isEqualTo(parsedConfig.getName());
         assertWithMessage("Property keys differ from expected.")
```

Comment:
can be removed since we check in for loop

Reply:
No. If the ```parsedConfig``` has a default attribute which is not allowed, this will fail while the next ones will pass.
For example,
```
/*
MissingOverride
javaFiveCompatibility = (default)false
nonExistentProperty = (default)checkstyle


*/

Reply:
but here we deal only with non default attributes:

```
assertWithMessage("Property keys differ from expected.")               
                .that(Arrays.asList(testConfig.getAttributeNames()))
                .containsAtLeastElementsIn(parsedConfig.getAttributeNames());
```

Reply:
> has a default attribute which is not allowed, this will fail while the next ones will pass.

currently, there is no way, how we can validate default values from input file

Reply:
Oh, yes. Should I then make a function to get all keys from input file?

Reply:
it won't help, we can't extract default properties from `DefaultConfiguration`, we will think about it later
lets just remove this assertion

Reply:
> we can't extract default properties from DefaultConfiguration

No, I meant to write a method in ```InputConfiguration``` to get the key set from both default and non-default attributes.

> lets just remove this assertion

done